src/tilemaker/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM buildpack-deps:xenial
+
+##### Install Node
+# Copied from https://github.com/nodejs/docker-node/blob/master/6.10/Dockerfile because we
+# need G++ 5 to successfully compile Mapnik but Node's standard Debian containers don't have it.
```

Comment:
I figured since the Node containers use [buildpack-deps](https://hub.docker.com/_/buildpack-deps/) `jessie` images it would be easiest to use the same thing but from a newer base image.  There might be alternatives, like upgrading g++ on the Node image or figuring out a way to get Mapnik installed from binaries such that the packages that use it won't try to compile it.  But this seemed easier than either of those, and made the crashes stop, so I went with this.